dataset/st_dataset.py

```diff
@@ -58,45 +62,45 @@ class SummDataset:
         * Query-based summarization
     """
 
-    def __init__(
-        self,
-        dataset_name: str,
-        description: str,
-        citation: str = None,
-        homepage: str = None,
-        huggingface_page: str = None,
-        train_set: Optional[Generator[SummInstance, None, None]] = None,
-        dev_set: Optional[Generator[SummInstance, None, None]] = None,
-        test_set: Optional[Generator[SummInstance, None, None]] = None,
-    ):
-        """
-        Following huggingface, the dataset contains train, dev and test set.
-        :param train_set:
-        :param dev_set:
-        :param test_set:
-
-        The following attributes should have been class attributes, however, python initialize all class variables
-            at the time of importation, and we don't want the dataset to be loaded once imported
-        :param dataset_name:
-        :param description:
-        :param is_query_based:
-        :param is_dialogue_based:
-        :param is_multi_document:
-        """
+    def __init__(self, dataset_args=None, splitseed=None):
+        """Create dataset information from the huggingface Dataset class"""
```

Comment:
can you add docstrings for the method args?

Reply:
Will do